homeassistant/components/sensor/homekit_controller.py

```diff
@@ -1,54 +1,71 @@
 """
-Support for Homekit koogeek sensors.
+Support for Homekit sensors.
 
 For more details about this platform, please refer to the documentation at
-https://home-assistant.io/components/sensor.koogeek/
+https://home-assistant.io/components/sensor.homekit_controller/
 """
 import logging
 
 from homeassistant.components.homekit_controller import (HomeKitEntity,
                                                          KNOWN_ACCESSORIES)
 
-from homeassistant.const import POWER_WATT_HOUR
-
 DEPENDENCIES = ['homekit_controller']
 
-KOOGEEK_CHARACTERISTICS = {
-    '4aaaf931-0dec-11e5-b939-0800200c9a66': 'realtime_energy',
-    '4aaaf932-0dec-11e5-b939-0800200c9a66': 'current_hour_data',
-    '4aaaf93f-0dec-11e5-b939-0800200c9a66': 'running_time'
+CHARACTERISTICS = {
+    '4aaaf931-0dec-11e5-b939-0800200c9a66': {
```

Comment:
I think that it might make sense to put this as a "device database" in the upstream lib (if possible). That way other consumers of that lib could also use it.